src/week1.org

```diff
@@ -0,0 +1,124 @@
+#+TITLE: Week 1
+#+AUTHOR: Kristaps Berzinch, Raymond Ortiz
+#+EMAIL: kristaps@robojackets.org
+
+* What are we doing today?
+- Trainer introductions
+- Overview of the RoboJackets software training program
+- Navigating and manipulating files and directories in the Linux terminal
+- Using Git and GitHub for version control
+
+* Who am I? - Kristaps
```

Comment:
Basically the way that reveal works is like this:

First overlay the slide contents on a specified presentation size (documented here: https://github.com/hakimel/reveal.js/#presentation-size). No resizing is done at this step, if it overflows, so be it (which is happening in this case). Next, resize the presentation window to actuallly fit on the target device. This means that as long as your slide contents are within the presentation size, you're good, but in our case, we're exceeding it, so it resizes things off the screen.

We can fix this by:
1. Increasing the margin, which I have done a bit
2. Increasing the presentation size, which gives us more room to work with (but makes everything smaller)
3. Decreasing the font size (which makes the text smaller, but keeps everything else the same).

When I was talking about the 1080p presentation size, I was talking about the 'export size' (which you can change in reveal.initialize, just for testing and playing around: https://github.com/hakimel/reveal.js/#presentation-size). 

When we do that, the slides can fit the full screen on a 1080p monitor, but when on a presentation screen, the slides get squashed (as the presentation size must be scaled down on the edges). 

Another solution that just occured to me would be to increase the presentation size without changing the aspect ratio. If you find a good presentation size, I'll go ahead and apply it to this slide deck.

1920x1080 margin 0.1
![2017-07-04-103245_1920x1080_scrot](https://user-images.githubusercontent.com/4349709/27839177-26a9641a-60a4-11e7-8198-6bfce37754dc.png)

1440x1050 margin 0.1 (x1.5 resolution of what we have). 
![2017-07-04-103428_1920x1080_scrot](https://user-images.githubusercontent.com/4349709/27839204-678fda22-60a4-11e7-938a-437f75175e0d.png)

Basically we can play around this to find a balance between readability and 'will the slides fit' for all the slides. 

Personally I think the defaults are pretty good (we should aim to limit things on a slide), but they can be annoying at times.